guides/common/modules/proc_creating-a-webhook-template.adoc

```diff
@@ -1,10 +1,11 @@
 [id="creating-a-webhook-template_{context}"]
 = Creating a Webhook Template
 
+Webhook templates are used to generate the body of HTTP request to {ProjectServer} when a webhook is triggered.
 Use the following procedure to create a webhook template in the {ProjectWebUI}.
 
 .Procedure
```

Comment:
There is a sentence `The webhook template can use a special variable called `@object` that can represent the main object of the event.` below that I'd like to add to. It's worth to mention, that there is also `@payload` variable always avaible that can be used to check what data are actually avaible since `@object` can be missing in case of certain events. Although we work on that, it's better to mention until this gets fixed.